airflow/ui/src/pages/Pools/PoolBar.tsx

```diff
@@ -0,0 +1,109 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { Box, Flex, Text, VStack } from "@chakra-ui/react";
+import { MdHourglassDisabled, MdHourglassFull } from "react-icons/md";
+
+import type { PoolResponse } from "openapi/requests/types.gen";
+import { Tooltip } from "src/components/ui";
+import { stateColor } from "src/utils/stateColor";
+
+type PoolBarProps = {
+  readonly pools: Array<PoolResponse>;
+};
+
+const PoolBar = ({ pools }: PoolBarProps) => (
+  <Flex direction="column" gap={4}>
+    {pools.map((pool) => {
+      // Calculate proportions
+      const totalSlots = pool.slots;
+      const openFlex = pool.open_slots / totalSlots || 0;
+      const scheduledFlex = pool.scheduled_slots / totalSlots || 0;
+      const runningFlex = pool.running_slots / totalSlots || 0;
+      const queuedFlex = pool.queued_slots / totalSlots || 0;
+      const occupiedFlex = pool.occupied_slots / totalSlots || 0;
+      const deferredFlex = (pool.include_deferred ? pool.deferred_slots / totalSlots : 0) || 0;
+
+      return (
+        <Box
+          borderColor="border.emphasized"
+          borderRadius={8}
+          borderWidth={1}
+          key={pool.name}
+          overflow="hidden"
+        >
+          <Flex alignItems="center" bg="bg.muted" justifyContent="space-between" p={4}>
+            <VStack align="start">
+              <Text fontSize="lg" fontWeight="bold">
+                {pool.name}
+              </Text>
+              {Boolean(pool.description) ? (
+                <Text color="gray.fg" fontSize="sm">
+                  {pool.description}
+                </Text>
+              ) : undefined}
+            </VStack>
+            <Tooltip
+              content={pool.include_deferred ? "Deferred Slots Included" : "Deferred Slots Not Included"}
+            >
+              {pool.include_deferred ? <MdHourglassFull size={25} /> : <MdHourglassDisabled size={25} />}
```

Comment:
```suggestion
              {pool.include_deferred ? <MdHourglassFull size={25} /> : <MdHourglassDisabled size={25} />}
```

This is way too big. Let's do a max of 18

Reply:
Done.

I have also made some changes.
1. Moved it beside the heading. Since right top will have the delete and edit icon.
2. Changed to clock icon, more relatable.
3. Removed for non-deferred task, icon shows only if deferred slots are included.

<img width="1632" alt="image" src="https://github.com/user-attachments/assets/ff7ef03c-bcd9-4f12-98c5-5ad3aaf0285a" />